lib/node/rpc.js

```diff
@@ -139,10 +139,9 @@ class RPC extends RPCBase {
         && cmd.method !== 'getblocktemplate'
         && cmd.method !== 'getbestblockhash') {
       this.logger.debug('Handling RPC call: %s.', cmd.method);
-      if (cmd.method !== 'submitblock'
-          && cmd.method !== 'getmemorypool') {
```

Comment:
`getmemorypool` is really old command that was replaced by `getrawmempool`.
I don't think bcoin ever supported it.

https://github.com/bitcoin/bitcoin/blob/1b6c463e033f861561d1a46ccf7eec069bbac09f/doc/release-notes/release-notes-0.7.0.md